### DISCUSSION TITLE
Upgrading existing install to 0.6.0.1 - maintain scripts, config, etc

### DISCUSSION THREAD

**Discussion by NeoMatrixJR** [mickem/nscp#832] [Q&A]
We're in the process of upgrading our Nagios Core VMs and we discovered they can no longer communicate with the NSClient++ agent we have deployed (v.0.5.2.41-x64).  As a possible remedy I'm testing upgrading to the latest "pre-release."  Is there a way to upgrade without having to uninstall/re-install/re-configure/re-add scripts that will still upgrade the SSL connection and use v4 packets?
also, preferably without rebooting?

**Comment by mickem** [comment]
Latest version should support NRPE 4
